Yona/Yona/Challenges/TimeFrameTimeZoneChallengeViewController.swift

```diff
@@ -75,19 +75,17 @@ class TimeFrameTimeZoneChallengeViewController: BaseViewController {
                 self.isFromButton = true
                 self.picker?.pickerTitleLabel.title = "From"
                 
-                if (self.activeIndexPath != nil) {
+                if self.activeIndexPath != nil {
```

Comment:
If possible, to be more coherent it is better to use the if let syntax every where.